Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM alpine:latest
+RUN adduser -D -u 1001 manifestuser
+RUN apk update && apk add --no-cache go git
+
+USER manifestuser
+WORKDIR /manifest
+COPY ./ ./
+
+RUN go install -buildvcs=false ./cmd/manifest
+ENV PATH="/home/manifestuser/go/bin:${PATH}"
+
+WORKDIR /app
+
+RUN git config --global --add safe.directory /app
+RUN echo "#!/bin/sh" > /home/manifestuser/start_cmd.sh
```

Comment:
If we need this, maybe we turn this into `script/ci`?

Ideally we'll build this into the app, though.

Reply:
Good idea. I will look into building it into the app as part of a separate change set.